core/inflation/src/lib.rs

```diff
@@ -75,6 +76,20 @@ fn inflate(x: Balance, years: u8) -> Option<Balance> {
 /// Reference(s):
 /// - <https://github.com/evolutionlandorg/bank/blob/master/contracts/GringottsBank.sol#L280>
 pub fn deposit_interest(amount: Balance, months: u8) -> Balance {
+	// The result of `((quot - 1) * precision + rem * precision / d)` is `197` when months is
+	// `12`.
+	//
+	// The default interest is `1_000`.
+	// So, we directly use `1_970_000` here instead `interest * 197 * 10^7`.
```

Comment:
Very confused about this comment. How the `1_970_000` and `* 10^7` calculated here? If we are do multiple operation with 197 and 1_000, the result should be 197_000 instead of 1_970_000. 

Both he explanation here and in the evolutionland repo is not clear enough.

Reply:
@hackfisher  Please help review this.

Reply:
Without the knowledge from https://github.com/evolutionlandorg/bank/blob/master/contracts/GringottsBank.sol#L280, you can't understand it.